generator_test.go

```diff
@@ -64,7 +62,7 @@ func TestProcessDumpFile(t *testing.T) {
 	conf.DefaultDBName = TestPostLocalDb
 	_ = DropDatabase(conf)
 	require.Nil(t, CreateDatabase(conf))
-	require.Nil(t, SQLCommandFile(conf, TestProcessDumpfile, false))
+	require.Nil(t, SQLCommandFile(conf, TestProcessDumpfile, true)) //Must ignore errors
```

Comment:
Must you?

Reply:
The issue here is that functions are pulled into the dump file which may already exist in the dump file. If we enable "exit on failure" we will never be able to import a full dump file unfortunately. Apparently adding  --schema= will include all systemwide stored procedures. We will need to look into why this is true. I will start another task to look into this, but for now it is outside the scope of this task.